### PR TITLE
[zh-cn]: update the translation of 'Author fast loading HTML pages'

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

finish translation

### Motivation

clear untranslated content

### CODE REVIEW

File: files/zh-cn/learn/html/howto/author_fast-loading_html_pages/index.md [hunk 1]
```diff
@@ -1,41 +1,43 @@
...
 ### 减小页面的大小
 
-直至今日，页面的大小仍是页面加载性能的最重要因素。
+目前，页面的大小仍是页面加载性能的最重要因素。
 
-通过压缩——排除不必要空格，注释，和将脚本、CSS 放入外部文件等减小页面的大小，可以在页面结构改变很小的情况下提高下载性能。
+通过消除不必要的空白和注释（通常称为压缩）以及将内联脚本和 CSS 移至外部文件来减少页面大小，可以提高下载性能，而无需对页面结构进行其他改动。
 
-诸如 [HTML Tidy](http://tidy.sourceforge.net/) 这类的工具可以从有效的 HTML 源文件中自动截去行首空格和额外的空行，其他工具则可以通过重新格式化源代码来压缩 JavaScript 或者通过混淆源码将长标识符替换为短标识符来减小文件大小。
+诸如 [HTML Tidy](https://www.html-tidy.org) 这类的工具可以从有效的 HTML 源文件中自动截去行首空格和额外的空行，其他工具则可以通过重新格式化源代码来“压缩” JavaScript 或者通过混淆源码将长标识符替换为短标识符来减小文件大小。
```
Comment (jasonren0403) [inline files/zh-cn/learn/html/howto/author_fast-loading_html_pages/index.md:22]:
```suggestion
诸如 [HTML Tidy](https://www.html-tidy.org) 这类的工具可以从有效的 HTML 源文件中自动截去行首空格和额外的空行，其他工具则可以通过重新格式化源代码来“压缩”JavaScript 或者通过混淆源码将长标识符替换为短标识符来减小文件大小。
```

File: files/zh-cn/learn/html/howto/author_fast-loading_html_pages/index.md [hunk 2]
```diff
@@ -47,38 +49,38 @@ Further reading:
...
 
-页面中任何具有动态特性的资源需要在页面被完全加载后才可以使用，所以最好在初始化时关闭动态特性（disable dynamic features），等页面加载完后再打开它。这样 JavaScript 就会在网页内容之后才加载，有助于提升页面加载的整体表现。
+任何需要在页面加载完成后才能使用的动态功能，都应先禁用，然后在页面加载完成后再启用。这将使 JavaScript 在页面内容之后加载，从而改善页面加载的整体外观。
 
 ### 减少内联脚本的数量
 
-内联脚本在页面加载过程中消耗很多资源，因为解析器认为内联脚本会改变页面结构。通常，尽量少的使用内联脚本和减少用`document.write()`来输出内容，在一定情况下可以加速整体页面的载入。现在浏览器中一般使用现代的 W3C DOM 方法操作页面内容，优于使用`document.write()`的传统方法。
+内联脚本对页面加载的影响很大，因为解析器必须假定内联脚本可能会在解析过程中修改页面结构。减少内联脚本的使用，尤其是减少使用 `document.write()` 来输出内容，可以提高页面的整体加载速度。为现代浏览器使用现代 [AJAX](/zh-CN/docs/Web/Guide/AJAX) 方法来操作页面内容，而不是使用基于 `document.write()` 的旧方法。
 
 ### 使用现代 CSS 和合法标记
```
Comment (yin1999) [inline files/zh-cn/learn/html/howto/author_fast-loading_html_pages/index.md:73]:
```suggestion
### 使用现代 CSS 和有效标记
```

File: files/zh-cn/learn/html/howto/author_fast-loading_html_pages/index.md [hunk 3]
```diff
@@ -88,79 +90,101 @@ Further reading:
...
 ```css
 table-layout: fixed;
 ```
 
-用 [`<col>`](/zh-CN/docs/Web/HTML/Element/col) 和 [`<colgroup>`](/zh-CN/docs/Web/HTML/Element/colgroup) 元素来指定列宽。
+并且用用 [`<col>`](/zh-CN/docs/Web/HTML/Element/col) 和 [`<colgroup>`](/zh-CN/docs/Web/HTML/Element/colgroup) 元素来指定列宽。
+
+### 对图片使用懒加载
+
+默认情况下，图片是**立即**加载的；也就是说，一旦图片在 HTML 中被处理，就会立即获取并渲染。所有立即加载的图片都会在文档的 {{domxref("Document.load_event", "load")}} 事件发送之前得到渲染。如果改用懒加载图像，浏览器就会暂缓加载图像，直到需要绘制{{Glossary("visual viewport","可视视口")}}时再加载。
```
Comment (yin1999) [inline files/zh-cn/learn/html/howto/author_fast-loading_html_pages/index.md:137]: mdn/content#29739

```suggestion
默认情况下，图片是**立即**加载的；也就是说，一旦图片在 HTML 中被处理，就会立即获取并渲染。所有立即加载的图片都会在文档的 [`load`](/zh-CN/docs/Web/API/Window/load_event) 事件发送之前得到渲染。如果改用懒加载图像，浏览器就会暂缓加载图像，直到需要绘制{{Glossary("visual viewport","可视视口")}}时再加载。
```

File: files/zh-cn/learn/html/howto/author_fast-loading_html_pages/index.md [hunk 4]
```diff
@@ -88,79 +90,101 @@ Further reading:
...
+      尽量减少文件数量以提高性能，同时将无关的 JavaScript 保存在单独的文件中以方便维护。
 
 ## 参见
 
-- 书籍：["Speed Up Your Site" by Andy King](http://www.websiteoptimization.com/)
-- 非常棒、讲的很全 [Best Practices for Speeding Up Your Web Site](http://developer.yahoo.com/performance/rules.html) (Yahoo!)
-- 用来分析和优化的工具：[Google PageSpeed Tools](https://developers.google.cn/speed)
+- 书籍：Andy King 撰写的[让你的站点访问更快](http://www.websiteoptimization.com/)
+- 非常棒、讲的很全的[加快站点访问的最佳实践](https://developer.yahoo.com/performance/rules.html)资料（来自 Yahoo!）
+- 用来分析和优化的工具：[Google PageSpeed 工具](https://developers.google.cn/speed?hl=zh-cn)
```
Comment (yin1999) [inline files/zh-cn/learn/html/howto/author_fast-loading_html_pages/index.md:190]: I think it's ok to remove the language tag.

```suggestion
- 用来分析和优化的工具：[Google PageSpeed 工具](https://developers.google.cn/speed)
```

**Review by yin1999 — COMMENTED** [review]
Sorry for the late. Here are my suggestions

File: files/zh-cn/learn/html/howto/author_fast-loading_html_pages/index.md [hunk 5]
```diff
@@ -1,41 +1,43 @@
...
 
 页面加载性能的优化不仅仅是针对那些带宽有限的拨号上网或移动设备用户需要看的内容，对于提供给宽带用户访问的内容同样重要并且可以导致巨大的提升，哪怕对于那些拥有最快网速的访客。
 
-## Tips
+## 技巧
 
 ### 减小页面的大小
 
-直至今日，页面的大小仍是页面加载性能的最重要因素。
+目前，页面的大小仍是页面加载性能的最重要因素。
```
[the remark this answers is not in the document]
Comment (jasonren0403) [reply files/zh-cn/learn/html/howto/author_fast-loading_html_pages/index.md:18]:
```suggestion
直至今日，页面的大小仍是页面加载性能的最重要因素。
```

**Review by yin1999 — APPROVED** [review]
Thank you @jasonren0403

**Pull request merged**